src/main/scala/dpla/ingestion3/mappers/providers/SdMapping.scala

```diff
@@ -0,0 +1,22 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.mappers.utils._
+import dpla.ingestion3.model._
+import org.json4s._
+
+
+// FIXME Why is the implicit conversion not working for JValue when it is for NodeSeq?
+class SdMapping extends MdlMapping with JsonMapping with JsonExtractor with IdMinter[JValue] {
+
+  // ID minting functions
+  override def useProviderName: Boolean = true
+
+  override def getProviderName: String = "sd"
+
+  override def agent = EdmAgent(
```

Comment:
Just noticed that `agent` is redundant and could be removed since it isn't actually overriding anything